packages/p/poco/xmake.lua

```diff
@@ -16,6 +16,18 @@ package("poco")
     add_versions("1.12.5", "92b18eb0fcd2263069f03e7cc80f9feb43fb7ca23b8c822a48e42066b2cd17a6")
     add_versions("1.13.3", "9f074d230daf30f550c5bde5528037bdab6aa83b2a06c81a25e89dd3bcb7e419")
 
+    if is_plat("mingw") then
+        add_patches("1.11.0", "patches/1.11.0/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.11.1", "patches/1.11.1/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.11.6", "patches/1.11.6/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.11.8", "patches/1.11.8/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.12.1", "patches/1.12.1/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.12.2", "patches/1.12.2/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.12.4", "patches/1.12.4/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
+        add_patches("1.12.5", "patches/1.12.5/cmake.patch", "8b4c1abc1fc635229333632a36c3f7850f2de70b9ba91db06d7c9aebcecc5ef1")
```

Comment:
if we use same patch file, we need only add single patch file, `"patches/1.12.5/cmake.patch"`

then `add_patches(">=1.11.0 <=1.12.5", "patches/1.12.5/cmake.patch")`

add_patches support semver.